OpenOversight/app/templates/auth/email/change_password.html

```diff
@@ -0,0 +1,11 @@
+<p>Dear {{ user.username }},</p>
+<p>Your password has just been changed.</p>
+<p>If you initiated this change to your password, you can ignore this email.</p>
+<p>
+  If you did not reset your password, please contact <a href="mailto:{{ help_email }}">the OpenOversight help account</a>; they will help you address this issue.
+</p>
+<p>Sincerely,</p>
+<p>The OpenOversight Team</p>
```

Comment:
Meta question: who is the OO team? Like say I deploy OO - Detroit how would the end user know that this email was coming from whoever deployed OO-Detroit and not the people who wrote this message.

Reply:
No clue, tbh. This is the standard farewell we use within our emails. I'd presume that the user would think that it's the OO team for that given city.